Jenkinsfile

```diff
@@ -11,34 +11,35 @@ pipeline {
     }
 
     environment {
-        IMAGE_REPO="cfpb/cfgov-python"
-        IMAGE_TAG="${JOB_BASE_NAME}-${BUILD_NUMBER}"
+        IMAGE_REPO = "cfpb/cfgov-python"
+        IMAGE_TAG = "${JOB_BASE_NAME}-${BUILD_NUMBER}"
         STACK_PREFIX = 'cfgov'
         NOTIFICATION_CHANNEL = 'cfgov-deployments'
     }
 
     parameters {
         booleanParam(
-            name: 'DEPLOY',
-            defaultValue: false,
-            description: 'Deploy the stack?'
+                name: 'DEPLOY',
+                defaultValue: false,
+                description: 'Deploy the stack?'
         )
         booleanParam(
-            name: 'REFRESH_DB',
-            defaultValue: false,
-            description: 'Refresh the database?'
+                name: 'REFRESH_DB',
+                defaultValue: false,
+                description: 'Refresh the database?'
```

Comment:
Could we not increase the indentation here, please?

Reply:
Sorry, this is my auto-format in idea.  Will fix.